modules/memory_bridge/layer.py

```
@@ -0,0 +1,46 @@
+            ``timestamp`` equal to or later than ``since`` are returned.
+
+        Returns
+        -------
+        list[MemoryEntry]
+            Matching entries in chronological order.
+        """
+
+        if since is None:
+            return self.all()
+        if since.tzinfo is None or since.tzinfo.utcoffset(since) is None:
+            raise ValueError("'since' must be timezone-aware.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)